composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "projet5/blog",
+    "description": "Blog PHP and Twig",
+    "require": {
+        "twig/twig": "v2.11.3"
+    },
+    "authors": [
+        {
+            "name": "Frédéric Vanmarcke",
+            "email": "fvanmarcke72@gmail.com"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Projet5\\": "src/"
```

Comment:
Ton autoload pointe vers le dossier src/ mais je n'ai pas vu de dossier src dans ton projet sauf erreur de ma part.

Reply:
Ce dossier est pour l'instant vide.
Il sera visible lors de l'issue "Add file required for routing"